cmd/minikube/cmd/start.go

```diff
@@ -126,6 +124,7 @@ func runStart(cmd *cobra.Command, args []string) {
 
 	var host *host.Host
 	start := func() (err error) {
+		fmt.Printf("Starting local OpenShift instance using '%s' hypervisor...\n", config.VMDriver)
```

Comment:
>  I am not able to think what approach to use to write any test case for above change. Let me know @hferentschik , if you have any approach.

This is very much testable :-), however, I think in this case it is not worth the effort. 

One way to test this, is to simply capture the stdout and make sure the log statement comes out. Eventually we will need some helper utility for that. For now we should be good.